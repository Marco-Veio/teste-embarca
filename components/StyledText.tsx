import { Text, TextProps } from "./Themed";

export function StyledText(props: TextProps) {
  return (
    <Text
      {...props}
      style={[
        props.style,
        {
          fontFamily: "starjedi",
          color: "black",
          textShadowColor: "gold",
          textShadowRadius: 2,
        },
      ]}
    />
  );
}
