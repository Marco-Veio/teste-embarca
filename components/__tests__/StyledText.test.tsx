import React from "react";
import renderer from "react-test-renderer";

import { StyledText } from "../StyledText";

describe("StyledText component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<StyledText />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
