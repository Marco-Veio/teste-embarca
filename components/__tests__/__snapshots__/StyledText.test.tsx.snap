// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledText component renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "color": "black",
          "fontFamily": "starjedi",
          "textShadowColor": "gold",
          "textShadowRadius": 2,
        },
      ],
    ]
  }
/>
`;
